### DISCUSSION TITLE
Preloading icons

### DISCUSSION THREAD

**Discussion by JasonSettings** [flet-dev/flet#4367] [Q&A] [answered]
### Question

Hello, fellow fletchers!
I have a minor issue I'd like to fix if possible. I got several themes for my small app available for selection via dropdown.
I got an app logo that is also changing colors along with the theme. I just manually made several icons for this.
So the issue is that the icon flickers for a moment as the theme change happens.
If I click through all my themes once, the image flickering no longer happens if I change the theme again after this.
So I was thinking that I should make some kind of image preload happen. I tried to do cycling through all the icons at app startup, but it did not help.
Maybe someone could help me come up with a working preload solution, or maybe there is a better fix for this issue?
Thanks in advance!

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Can you try setting [ `Image.gapless_playback`](https://flet.dev/docs/controls/image/#gapless_playback) to `True`.

**Comment by JasonSettings** [comment]
Thank you! This fixes the issue.
